editor/src/components/inspector/fill-hug-fixed-control.tsx

```diff
@@ -209,30 +210,52 @@ export const FillHugFixedControl = React.memo<FillHugFixedControlProps>((props)
 
   const onAdjustHeight = React.useCallback(
     (value: number | EmptyInputValue) => {
-      if (typeof value === 'number') {
+      if (typeof value !== 'number') {
+        return
+      }
+      if (heightCurrentValue?.type === 'fill') {
+        executeFirstApplicableStrategy(
```

Comment:
Minor thing: You could make a utility function which calls this with a given strategy.